docker/Dockerfile.bionic_debug

```diff
@@ -16,7 +16,7 @@ RUN apt-get update && \
       tcpdump \
       net-tools \
       lsof \
-      sudo && \
+      sudo=1.8.16-0ubuntu1.6 && \
```

Comment:
Maybe I'm mistaken, but won't pinning prevent us from picking up the next new version. Isn't it sufficient to do an apt-get update and the pulley's the latest updated packages?

Reply:
@ericvn you are right about how pinning packages to versions work. I guess there are 2 sides to  it - Pinning to a known version has the plus side of using a stable version we know is good and tested, the flip side is what you mentioned. apt-get update alone does not install the latest packages. We need to combine apt-get update along with apt-get install to get the latest package versions. These commands were already in the original Dockerfile.xenial_debug. So the 2 changes I added were to clear the apt-cache and pin the version.

Reply:
@johnma14 I agree either can be used. Depends on if one wants to automate picking up new packages or if they should be pinned (it doesn't make sense to only pin one in my mind though). But Steve mentions a process and that may or may not have a specific answer to this problem.

Reply:
When I tested this in my env, I couldn't get the same security vulnerability that is mentioned in the PR. That is because I was creating a new image and that automatically pulled the latest version of curl, so the images built using that base image was fine. In the 'docker build' command, unless you specify the --pull option, it will use a local image from the local repository if one is available. I suspect that is what is happening with the formal images that we are pushing out there which is why we are seeing them using the older versions. Again I do not know what is the process of building these images and cleaning them after pushing these images out to the hub (which is why I hesitated to make this change in the makefile that builds istio components). This might be a good read (https://www.databasesandlife.com/docker-build-pull-option/). I made these similar changes to the build command for the docker images we build for the book info samples.

  We could do that here or the other option is to pin versions. I agree that if we choose to pin versions, it makes sense to pin versions for all the other packages as well. I was hoping to get a feedback before doing this on all packages. Either way I will go ahead and open a CVE report like @sdake mentioned. 

Thanks @ericvn, appreciate your thoughts.

Reply:
It seems to me pining the packages to stable versions makes more sense. Because:
1. These packages are stable feature-wise, we don't need to catch up the versions frequently. 
2. Better security with stable versions. Avoid the noise of new bugs in the new versions.

Reply:
@johnma14 Thanks for the change. But the current stable sudo version is 1.8.27, according to https://www.sudo.ws/stable.html. Are you sure we want to downgrade it to 1.8.16?

Reply:
@myidpt Makes sense to me. I will go ahead and pin versions for the other packages as well if that makes sense. Appreciate your feedback.

Reply:
@liminw The current stable version like you said should be good as well (I should have checked what was the latest release, my bad. Thanks for pointing it out). I guess the problem is that our process of building these images are not correctly pulling the latest versions of these packages even though we have the apt-get commands to update-install-upgrade these packages. One explanation could be that a previously build base image is available in the local repository where these images are being built and it isn't removed. So all the istio component images are built using this.

Reply:
@johnma14 Makes sense. Thanks again for fixing all these issues!